src/app/(dashboard)/dashboard/advisor-application/page.tsx

```diff
@@ -48,51 +48,66 @@ const getBase64 = (img: FileType, callback: (url: string) => void) => {
   reader.readAsDataURL(img);
 };
 // Allowed image MIME types
-const allowedTypes = ["image/jpeg", "image/png", "image/gif", "image/webp"];
+const allowedTypes = [
+  "image/jpeg",
+  "image/jpg",
+  "image/png",
+  "image/gif",
+  "image/webp",
+];
 
 // Max file size (5MB in bytes)
-const maxSize = 5 * 1024 * 1024;
+const maxSize = 1 * 1024 * 1024;
 
 export default function AdvisorApplicationPage() {
+  const { user } = useUser();
+
+  console.log(user, "user");
+
+  const router = useRouter();
   const { advisorType } = useDashboard();
+
+  const [messageApi, contextHolder] = message.useMessage();
   const [form] = Form.useForm<CareerAdvisorApplicationInfo>();
   const { setFieldValue, resetFields } = form;
   const [imageUrl, setImageUrl] = useState<string | undefined>(undefined);
   const { post } = useApiClient();
 
   const handleSubmit: FormProps<CareerAdvisorApplicationInfo>["onFinish"] =
     async (values) => {
-      const payload = {
-        careerAdvisor: {
-          ...values,
-        },
-      };
-      const formDataItem = convertToFormData(payload);
-      await post("/api/career-advisors", formDataItem).then(() => {
+      const formData = new FormData();
+
+      formData.append("CareerAdvisor.Name", JSON.stringify(values.name));
+      formData.append("CareerAdvisor.Title", JSON.stringify(values.title));
+      formData.append(
+        "CareerAdvisor.PhoneNumber",
+        JSON.stringify(values.phoneNumber)
+      );
+      formData.append(
+        "CareerAdvisor.Qualification",
+        JSON.stringify(values.qualifications)
+      );
+      formData.append(
+        "CareerAdvisor.MotivationStatement",
+        JSON.stringify(values.motivationStatement)
+      );
+      formData.append(
+        "CareerAdvisor.expertise",
+        JSON.stringify(values.expertise)
+      );
+      formData.append(
+        "CareerAdvisor.Expertise",
+        JSON.stringify(values.expertise)
+      );
+
+      await post("/api/career-advisors", formData).then(() => {
         resetFields();
         setImageUrl(undefined);
+        router.push("/dashboard");
       });
     };
 
```

Comment:
No need to use callback since you're already using `async/await`
```suggestion

      await post("/api/career-advisors", formData);
      resetFields();
      setImageUrl(undefined);
      router.push("/dashboard");
```